package.json

```diff
@@ -66,11 +73,11 @@
   "repository": "https://github.com/tapjs/node-tap.git",
   "scripts": {
     "snap": "bash scripts/snap.sh",
-    "test": "node bin/run.js test -M coverage-map.js",
+    "test": "node bin/run.js test/**/**.js test/*.js -M coverage-map.js",
```

Comment:
I worry about using glob patterns on CLI.  Why not alter the `test-ignore` pattern in the `tap` section of package.json?

Even if we use these patterns these seem a bit wrong.  I think `test/**/*.js` should cover everything?  I think the patterns would need to be quoted if passed via CLI to prevent inappropriate processing by the shell, I'm not sure if the cross-platform way to quote them would be single-quotes `'` or double quotes `"`.